packages/firestore/src/local/indexeddb_remote_document_cache.ts

```diff
@@ -95,66 +96,62 @@ export class IndexedDbRemoteDocumentCache implements RemoteDocumentCache {
   }
 
   /**
-   * Adds the supplied entries to the cache. Adds the given size delta to the cached size.
+   * Adds the supplied entries to the cache.
+   *
+   * Callees must ensure that the total document size in the cache is updated
+   * after all entries are written (via `updateMetadata()`).
```

Comment:
Callees => Callers

Also, not sure if it's worth doing in this PR, but I think if we made IndexedDbRemoteDocumentChangeBuffer into a ["nested class"](https://stackoverflow.com/a/32494175) of IndexedDbRemoteDocumentCache, then we could make addEntry() [and potentially removeEntry(); see my comment on removeOrphanedDocuments] into private methods only called by applyChanges().  Then the size accounting is very well encapsulated in RemoteDocumentCache/RemoteDocumentChangeBuffer with callers being oblivious, and that seems kinda' great.

Reply:
It might not have been worth it, but it is done. It solves some of the data encapsulation issues in this class hierarchy.